configs/dev-comm.yaml

```diff
@@ -0,0 +1,9 @@
+keri:
+  witness: backer #pools
```

Comment:
Lets name this `backerType` in both files. And name it "ledger" for backer.

Lets remove `pools` from this file as it's unused. Lets keep configs as minimal as possible. And remove the `backer` section from `dev-ent.yaml`